addons/block-cherry-picking/addon.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Block cherry-picking",
+  "description": "Adds the ability to drag a block out of the middle of a script (instead of the entire stack) while holding the Ctrl key.",
+  "info": [
+    {
+      "type": "notice",
+      "text": "On macOS, use the Cmd key instead of the Ctrl key.",
+      "id": "macContextDisabled"
+    }
+  ],
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang"
```

Comment:
Your link in custom-block-shape points to your Scratch profile instead. Would probably be best to pick one and be consistent